From 0abc105a66b514ac444aff3544b76ad9fa8863fe Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 11 Mar 2022 12:27:27 +0800
Subject: [PATCH 076/274] LF-6254-2 arm64: dts: imx93: Add "simple-mfd"
 compatible string to media_blk_ctrl node

Since media block controller contains registers to control
MIPI DPHY, the media_blk_ctrl node can be the parent
device node of the MIPI DPHY node.  Add "simple-mfd" compatible
string to media_blk_ctrl node so that the child device
can be populated.

Cc: Sandor Yu <Sandor.yu@nxp.com>
Reviewed-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Liu Ying <victor.liu@nxp.com>
---
 arch/arm64/boot/dts/freescale/imx93.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/imx93.dtsi b/arch/arm64/boot/dts/freescale/imx93.dtsi
index ae6ccdf20..005148e3d 100644
--- a/arch/arm64/boot/dts/freescale/imx93.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx93.dtsi
@@ -728,7 +728,7 @@ sentnl_mu: sentnl-mu {
 		};
 
 		media_blk_ctrl: blk-ctrl@4ac10000 {
-			compatible = "fsl,imx93-media-blk-ctrl", "syscon";
+			compatible = "fsl,imx93-media-blk-ctrl", "syscon", "simple-mfd";
 			reg = <0x4ac10000 0x10000>;
 			power-domains = <&mediamix>;
 			clocks = <&clk IMX93_CLK_MEDIA_APB>,
-- 
2.25.1

